src/pyfmi/fmil_import.pxd

```diff
@@ -56,18 +55,18 @@ cdef extern from 'fmilib.h':
     ctypedef unsigned int fmi1_value_reference_t
     ctypedef char   fmi1_boolean_t
     ctypedef int    fmi2_boolean_t
-    ctypedef void * fmi1_component_t
-    ctypedef void * fmi2_component_t
-    ctypedef char * fmi1_string_t
-    ctypedef char * fmi2_string_t
+    ctypedef void*  fmi1_component_t
+    ctypedef void*  fmi2_component_t
+    ctypedef char*  fmi1_string_t
+    ctypedef char*  fmi2_string_t
     ctypedef int    fmi1_integer_t
     ctypedef int    fmi2_integer_t
-    ctypedef long unsigned int size_t
-    ctypedef void * jm_voidp
-    ctypedef char * jm_string
+    # ctypedef long unsigned int size_t # same as via stdlib.h
```

Comment:
This gave quite a few warnings.

Reply:
I wonder if we should just remove it, we will always have commit history and perhaps we should just be less conservative in the clean-up approach? Feel free to ignore and resolve.

Reply:
I think these (also others lower down) are good to keep around, to be able to see "This exists in FMIL, but is unused right now".